definitions/scenarios/upgrade_to_capsule_6_8.rb

```diff
@@ -27,6 +27,7 @@ class PreUpgradeCheck < Abstract
     def compose
       add_steps(find_checks(:default))
       add_steps(find_checks(:pre_upgrade))
+      add_step(Checks::Foreman::CheckpointSegments)
```

Comment:
Copying from a private conversation: on Capsules this isn't really relevant since there is no PostgreSQL until we introduce Pulp 3. That's not for 6.8. Future versions that do can assume a PG 12 or newer so this check isn't needed.

Reply:
@ekohl Yes, I agree. I will remove it.